thau-api/src/main/java/com/mgrin/thau/configurations/ThauConfigurations.java

```diff
@@ -0,0 +1,121 @@
+package com.mgrin.thau.configurations;
+
+import java.util.LinkedList;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.mgrin.thau.configurations.broadcast.HTTPBroadcastConfiguration;
+import com.mgrin.thau.configurations.jwt.JWTConfiguration;
+import com.mgrin.thau.configurations.strategies.FacebookStrategyConfiguration;
+import com.mgrin.thau.configurations.strategies.GoogleStrategyConfiguration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ThauConfigurations {
+
+    public enum Strategy {
+        FACEBOOK("facebook"),
+        GOOGLE("google"),
+        PASSWORD("password");
+
+        private String value;
+        Strategy(String value) {
+            this.value = value;
+        }
+
+        @JsonValue
+        public String getValue() {
+            return value;
+        }
+    }
+
+    public enum BroadcastChannel {
+        HTTP("http"),
+        KAFKA("kafka");
+
+        private String value;
+        BroadcastChannel(String value) {
+            this.value = value;
+        }
+
+        @JsonValue
+        public String getValue() {
+            return value;
+        }
+    }
+
+    @Value("${thau.env}")
+    @NonNull
+    private String environment;
+
+    @Value("${thau.app.name}")
+    @NonNull
+    private String appName;
+
+    @Value("${thau.strategies.password.enabled}")
+    private boolean isPasswordStrategyEnabled;
+
+    @Autowired
+    private JWTConfiguration jwtConfiguration;
+
+    @Autowired(required = false)
+    private GoogleStrategyConfiguration googleStrategyConfiguration;
+
+    @Autowired(required = false)
+    private FacebookStrategyConfiguration facebookStrategyConfiguration;
+    
+    @Autowired(required = false)
+    private HTTPBroadcastConfiguration httpBroadcastConfiguration;
+    
+    public String getEnvironment() {
+        return environment;
+    }
+
+    public String getAppName() {
+        return appName;
+    }
+    public BroadcastChannel[] getBroadcastChannels() {
+        LinkedList<BroadcastChannel> broadcastChannels = new LinkedList<>();
```

Comment:
```suggestion
        List<BroadcastChannel> broadcastChannels = new LinkedList<>();
```
You generally declare with the interface.